src/components/InstructionsEditor.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {t} from 'i18next';
+
+export default function InstructionsEditor({instructions, onCancelEditing}) {
+  return (
+    <div className="instructions-editor">
+      <div className="instructions-editor-menu">
```

Comment:
Suggest to follow BEM naming rules like the rest of the selectors.
`"instructions-editor-menu"` to `"instructions-editor__menu"`